pkg/app/src/components/Account/modules/collectables-section/collectables-list.tsx

```diff
@@ -0,0 +1,38 @@
+import { Grid } from '@mui/material'
+import React, { FC, memo } from 'react'
+
+import CollectableCard from './collectables-card'
+import LoadingCollectableCard from './loading-collectable-card'
+import { CollectablesForUserQuery } from '@gamedao-haiku/graphql/dist'
+import { Collectable } from 'src/@types/collectable'
+
+interface CollectablesListProps {
+	loading: boolean
+	items: CollectablesForUserQuery
+}
+
+const CollectablesList: FC<CollectablesListProps> = ({ loading, items }) => {
+	if (!Array.isArray(items?.nfts) && !loading) return null
+
+	return (
+		<Grid container sx={{ pt: 3 }} spacing={{ xs: 1, md: 0.5 }} columns={{ xs: 2, sm: 8, md: 12 }}>
+			{loading ? (
+				[1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map((x) => (
```

Comment:
why 10? not sure if this is really cool, especially if you show 10 for loading and you get 1 as a result. there should be a better way.

Reply:
do you prefer reducing them to 5 or 2?

Reply:
It is currently displays 5 per row so if in the load state we display 5 it can be  better and then get 1 as a result it is better than 10 since in the loading I can't know how many is there unless you have a more efficient way you prefer